src/poliastro/twobody/orbit.py

```diff
@@ -228,6 +229,41 @@ def from_vectors(cls, attractor, r, v, epoch=J2000, plane=Planes.EARTH_EQUATOR):
         ss = RVState(attractor, r, v)
         return cls(ss, epoch, plane)
 
+    @classmethod
+    def from_skycoord(cls, attractor, coord):
+        """Creates an `Orbit` from an attractor and astropy `SkyCoord` instance.
+        This method accepts `SkyCoord` in any reference frame unlike `Orbit.from_vector`
+        which can accept inputs in only inertial reference frame centred at attractor.
+        Also note that the frame information is lost after creation of the orbit and
+        only the inertial reference frame at body centre will be used for all purposes.
+        Parameters
+        ----------
+        attractor: Body
+            Main attractor
+        coord: astropy.coordinates.SkyCoord
+            Position and velocity vectors in any reference frame. Note that `SkyCoord` must have
+            a representation and its differential with respect to time.
+
+
+        """
+        if not isinstance(coord, SkyCoord):
```

Comment:
I would remove the `isinstance` check. It's unpythonic (we should rely on "duck typing" instead) and also it's super slow.

Reply:
Sure, removed